luminaire/model/window_density.py

```diff
@@ -704,6 +712,7 @@ def _anomalous_region_detection(self, input_df=None, window_length=None,
         :param luminaire.model.lad_structural.LADStructuralModel agg_data_model: Prediction model for aggregated data.
         :param luminaire.model.window_density.WindowDensityModel past_model: Past stored window density model.
         :param str detection_method: Selects between "kldiv" or "sign_test" distance method.
+        :param agg_data: Aggregated Data per day.
```

Comment:
Same comment as before

Reply:
This is a higher level aggregated data which is per day at this moment